client/src/res/values/joyride.js

```diff
@@ -50,18 +50,30 @@ export const JOYRIDE_STEPS = [
           target: '.leaflet-states-pane .leaflet-interactive:nth-child(4)',
           content: 'Click on California to unlock it and then click "Next"',
           spotlightClicks: true,
+          disableBeacon:true,
+        },// At the end of step 7, need to jump zoom to la county
+        { //8
+          target: '.leaflet-counties-pane .leaflet-interactive:nth-child(240)',
+          content: 'Incidents can also be viewed at a county level with the same functionality as states. Zoom in to see data for individual counties.',
+          spotlightClicks: true,
+          disableBeacon: true,
+        },
+        {// 9
+          target: '.MuiToggleButtonGroup-root',
```

Comment:
Getting a lot of **Uncaught TypeError: Cannot read property 'isSameNode' of undefined at getScrollParent**. Possible [dependency error on @material-ui/core](https://github.com/gilbarbara/react-joyride/issues/575#issuecomment-617750898)?

Reply:
I tried changing this to map-bar and it still threw the error. The errors comming from this snippet:
` var parent = scrollParent(element);

  if (parent.isSameNode(scrollDoc())) {
    if (forListener) {
      return document;
    }

    return scrollDoc();
  }`

Not sure why "getScrollParent" isn't returning anything for the leaflet control div with the map toggle. Do you think this is something I can just ignore or should I try to find a different way to select the toggle that will fix this?